guides/common/modules/proc_configuring-selinux-to-permit-access-to-repos-on-custom-ports.adoc

```diff
@@ -0,0 +1,25 @@
+[id="configuring-selinux-to-permit-access-to-repositories-on-custom-ports_{context}"]
+= Configuring SELinux to Permit Access to Repositories on Custom Ports
+
+ifdef::satellite,foreman-el,katello[]
```

Comment:
Why this `ifdef`? If it is due to SELinux not being there on Debian then it should be done at another level. The whole procedure is about SELinux and doesn't apply to Debian. If Orcharhino runs without SELinux then again the whole procedure doesn't apply.

Reply:
Agree as in remove that line?

Reply:
trying that, removed it